motoko/actor_reference/README.md

```diff
@@ -0,0 +1,91 @@
+# Actor references
+
+![Compatibility](https://img.shields.io/badge/compatibility-0.7.0-blue)
+[![Build Status](https://github.com/dfinity/examples/workflows/motoko-actor_reference-example/badge.svg)](https://github.com/dfinity/examples/actions?query=workflow%3Amotoko-actor_reference-example)
+
+This example demonstrates a simple use of an [actor reference](https://sdk.dfinity.org/docs/language-guide/language-manual.html#exp-actor) to convert
+a textual representation of some principal to a value of an actor type, that, now typed
+as an actor, can be communicated with by calling the `shared` functions of its interface.
+
+Actor references should be used sparingly, and only when necessary, to provide a typed interface to a raw principal identifier.
+Using an actor reference is regarded as unsafe since it is effectively a promise, made by the programmer to the compiler, that the given principal obeys the given interface.
```

Comment:
Maybe find a differnet word than “promise”, as it might be confused with `async`-style promise?

Reply:
Maybe say something like "It's the programmers responsibility, as opposed to the compiler's, that the given principal obeys the given interface"?